primo/opt_model/result_parser.py

```diff
@@ -302,14 +302,15 @@ def __init__(self, wd: WellData, clusters_dict: dict, plugging_cost: dict):
         # for now include a pointer to well data, so that I have column names
         self.wd = wd
         self.projects = {}
+        self.clusters_dict = clusters_dict
 
         index = 1
-        for cluster, wells in clusters_dict.items():
-            self.projects[index] = Project(
+        for cluster, wells in self.clusters_dict.items():
+            self.projects[cluster] = Project(
                 wd=wd,
                 index=wells,
                 plugging_cost=plugging_cost[cluster],
-                project_id=index,
+                project_id=cluster,
             )

```

Comment:
This change is fine, but we need to keep in mind that the example notebook will now display cluster numbers instead of project numbers (i.e., instead of 1,2,3,4,5,6, the project ids will be 7,13,14,36,40,49,92). I am not sure who had strong opinions about this.